code/modules/mob/living/carbon/human/human.dm

```diff
@@ -592,6 +592,9 @@
 		say(pick(";RAAAAAAAARGH!", ";HNNNNNNNNNGGGGGGH!", ";GWAAAAAAAARRRHHH!", "NNNNNNNNGGGGGGGGHH!", ";AAAAAAARRRGH!" ), forced = "hulk")
 		if(..(I, cuff_break = FAST_CUFFBREAK))
 			dropItemToGround(I)
+	else if(dna.species.id == "memezombies")
```

Comment:
Make this a species trait instead.

Reply:
Hmkey!

Reply:
https://github.com/BeeStation/BeeStation-Hornet/assets/45698448/e6a6d7f4-3c84-4404-98c9-ea3fe02e51aa



Like this?